python/ray/tune/tests/test_ray_trial_executor.py

```diff
@@ -1,4 +1,6 @@
 # coding: utf-8
+from freezegun import freeze_time
```

Comment:
This is to mock the behavior of anything related to time.time() or time.monotonic() etc.
https://github.com/spulec/freezegun

I think it's a neat mock to have, especially since we have these time checks here and there in our codebase. A pretty common and popular construct for unit testing - available in multiple languages.

Reply:
Ok, let's keep it then